routes/archive.js

```diff
@@ -4,7 +4,7 @@ const getArchiveAndChildren = require('../lib/get-archive-and-children');
 module.exports = (elastic, config) => ({
   method: 'GET',
   path: '/documents/{id}/{slug?}',
-  handler: (request, reply) => reply(),
+  handler: (request, reply) => HTMLResponse(request, reply, elastic, config),
```

Comment:
No, I had a look and couldn't find an option either, that's why I did it like this. But yeah, by default it should now send html back, rather than an empty response